_schemaV2.graphql

```diff
@@ -2171,6 +2171,9 @@ type Artwork implements Node & Searchable & Sellable {
   # Based on artwork location verify that VAT info for the partner is complete.
   vatRequirementComplete: Boolean
 
+  # The visibility level of the artwork
+  visibilityLevel: String
+
```

Comment:
Curious what others think here of typing?

I wonder if it is helpful at all to specify an enum type here (well in the artwork.index page) similar to how it is handled in [artwork import source](https://github.com/artsy/metaphysics/blob/main/src/schema/v2/artwork/index.ts#L323-L335) for example. Then it is a little more explicit in the schema that the values for visibilityLevel are expected to only be X,Y,Z. Example [here](https://github.com/artsy/metaphysics/pull/3886/files)

Reply:
So I based this on how we handle `cost_currency_code` which is also an enum field on `Artwork` in gravity, but definitely open to changing it!

Reply:
blocking: +1 to adding an enum for this. Part of adding an enum to a graphql schema is that client will know that the field is an enum and not just a string. I can see why we might have done that for `cost_currency_code`, but honestly that should be a graphql enum as well.

Reply:
alrighty I'm convinced and I switched it to an ENUM!